localisation/00_wc_title_key.csv

```diff
@@ -182,14 +182,14 @@ temple_empire_of_loa_group;Empire of;
 # Common Magocracy;;;;;;;;;;;;;;x;;;;;;;
 magocracy_baron;Master;�������;;;;;;;;;;;;x;;;;;;;
 magocracy_count;Master;�������;;;;;;;;;;;;x;;;;;;;
-magocracy_duke;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
-magocracy_king;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
-magocracy_emperor;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
+magocracy_duke;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
+magocracy_king;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
+magocracy_emperor;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
 magocracy_baron_female;Master;�������;;;;;;;;;;;;x;;;;;;;
 magocracy_count_female;Master;�������;;;;;;;;;;;;x;;;;;;;
-magocracy_duke_female;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
-magocracy_king_female;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
-magocracy_emperor_female;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
+magocracy_duke_female;Great Master;������� �������;;;;;;;;;;;;x;;;;;;;
```

Comment:
Suggested Mistress, a feminine master. Can also use "Magistrix" in lieu of the Grand Magistrix of Suramar. In that case, you should use Grand Magister for the male titles.
```suggestion
magocracy_duke_female;Great Mistress;������� �������;;;;;;;;;;;;x;;;;;;;
```

Reply:
Going to rework the magocracy titles.
Here're ideas:

- Male Baron - Librarian
- Male Count - Archivist
- Male Duke - Magister
- Male King - Grand Magister
- Male Emperor - Grand Archmagister
- Female Baron - Librarian
- Female Count - Archivist
- Female Duke - Magistrix
- Female King - Grand Magistrix
- Female Emperor - Grand Archmagistrix

Comments are welcome.